lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -253,10 +217,9 @@ export default class BitcoinProcessor {
     console.info(`Fee: ${fee}. Anchoring string ${anchorString}`);
     const sidetreeTransactionString = `${this.sidetreePrefix}${anchorString}`;
 
-    const address = this.privateKey.toAddress();
-    const unspentOutputs = await this.bitcoinClient.getUnspentCoins(address);
+    const unspentOutputs = await this.bitcoinClient.getUnspentCoins();
 
-    let totalSatoshis = unspentOutputs.reduce((total: number, coin: Transaction.UnspentOutput) => {
+    let totalSatoshis = unspentOutputs.reduce((total: number, coin: BitcoinUnspentCoinsModel) => {
       return total + coin.satoshis;
     }, 0);
```

Comment:
Design: What we really want is an abstraction that simply returns the remaining balance left in the address. So change the interface method from `getUnspentCoins()` to something like `getBalance()`.

Reply:
Great idea .. will update.